playbooks/tests/tasks/lbaas.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: tests lbaas
+  hosts: controller[0]
+  vars:
+    image_name: cirros
+    flavor_name: m1.tiny
+    pub_net: external
+    pri_net: internal
+    sub_net: internal_v4
+    stack_name: test_lbaas
+  tasks:
+  - copy: src={{ lookup('env', 'URSULA_ENV') }}/../..//playbooks/tests/tasks/lb_stack.yml dest=/tmp/lb_stack.yml owner=ubuntu group=ubuntu
+  - name: make sure admin has no role - heat_stack_user
+    shell: . /root/stackrc; openstack role add --user admin --project admin heat_stack_user; openstack role remove --user admin --project admin heat_stack_user;
+  - name: create heat stack for lbaas
+    shell: . /root/stackrc; openstack stack create --template /tmp/lb_stack.yml --parameter "image={{ image_name }};flavor={{ flavor_name }};public_network={{ pub_net }};private_network={{ pri_net }};subnet={{ sub_net }}" --enable-rollback {{ stack_name }}; sleep 60
```

Comment:
Instead of just sleeping for 60 seconds, you could use the `--wait` option which will wait for the stack to reach completion or failure.